lua/lspconfig/configs.lua

```diff
@@ -34,13 +35,13 @@ end
 ---@param config_name string
 ---@param config_def table Config definition read from `lspconfig.configs.<name>`.
 function configs.__newindex(t, config_name, config_def)
-  validate {
-    name = { config_name, 's' },
-    default_config = { config_def.default_config, 't' },
-    on_new_config = { config_def.on_new_config, 'f', true },
-    on_attach = { config_def.on_attach, 'f', true },
-    commands = { config_def.commands, 't', true },
-  }
+  if nvim_eleven then
+    validate('name', config_name, 'string')
+    validate('default_config', config_def.default_config, 'table')
+    validate('on_new_config', config_def.on_new_config, 'function', true)
+    validate('on_attach', config_def.on_attach, 'function', true)
+    validate('commands', config_def.commands, 'table', true)
+  end
```

Comment:
This entire module will only be usfeul for Nvim older than 0.11, since `vim.lsp.config` makes this module obsolete. So this validation isn't relevant for Nvim 0.11

Reply:
reverting changes then.